xds/src/main/java/io/grpc/xds/internal/sds/ServerWrapperForXds.java

```diff
@@ -36,22 +42,59 @@
 public final class ServerWrapperForXds extends Server {
   private final Server delegate;
   private final XdsClientWrapperForServerSds xdsClientWrapperForServerSds;
+  @Nullable XdsServerBuilder.ErrorNotifier errorNotifier;
+  @Nullable XdsClientWrapperForServerSds.ServerWatcher myServerWatcher;
 
-  ServerWrapperForXds(Server delegate, XdsClientWrapperForServerSds xdsClientWrapperForServerSds) {
+  ServerWrapperForXds(
+      Server delegate,
+      XdsClientWrapperForServerSds xdsClientWrapperForServerSds,
+      @Nullable XdsServerBuilder.ErrorNotifier errorNotifier) {
     this.delegate = checkNotNull(delegate, "delegate");
     this.xdsClientWrapperForServerSds =
         checkNotNull(xdsClientWrapperForServerSds, "xdsClientWrapperForServerSds");
+    this.errorNotifier = errorNotifier;
   }
 
   @Override
   public Server start() throws IOException {
```

Comment:
`start()`  throws IllegalStateException if already started. Although current implementation does so but could be more graceful without side effect.

Reply:
Added a check similar to `ServerImpl` logic.